libzeth/test/blake2s_test.cpp

```diff
@@ -175,17 +175,42 @@ TEST(TestBlake2sComp, TestTrue)
             0, 1, 1, 1, 0, 0, 1, 0, // 72
             0, 1, 1, 0, 1, 1, 0, 0, // 6C
             0, 1, 1, 0, 0, 1, 0, 0, // 64
+            0, 0, 0, 0, 0, 0, 0, 0, // padding
+            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
         },
         ZERO);
 
     libsnark::block_variable<FieldT> input(
         pb, {pb_va_input}, "blake2s_block_input");
 
+    // default chaining value
+    libsnark::pb_variable_array<FieldT> pb_va_h = from_bits(
```

Comment:
what does `va` stand for in `pb_va_h`? (likewise for `pb_va_input`)

Reply:
it stand for pb variable. I called it like this to be consistent with pb_va_input.

Reply:
Ah right, would you mind renaming all `pb_va_*` to `pb_var_*? I think this would be clearer

Reply:
Sure. Updated.